pcp/columns/wchan

```diff
@@ -0,0 +1,9 @@
+# pcp-htop(1) configuration file - see pcp-htop(5)
+
+[wchan]
+caption = WCHAN
+width = 8
+description = PCP wchan description
```

Comment:
Let's improve the description.  Use "pminfo -t proc.psinfo.wchan" output, as a sentence (start with upper case letter).